### ISSUE TITLE
Refactoring deletes comments at the end of the file

### ISSUE THREAD

**Issue by komodovaran** [sourcery-ai/sourcery#138]
### Configuration
Sorcery Version: 0.9.0
Sourcery Plugin Version: 0.9.0
Platform: Windows 10; PyCharm 2021.1.1 (Professional Edition) Build #PY-211.7142.13

### Expected Behavior
Comments should be left alone.

### Actual Behavior
Comment is deleted when it's placed at the bottom of the file

### Steps to Reproduce
This

```
ls = []
for i in range(10):
    ls.append(i)

# This is a comment below ALL code

# And this

# And this too
```

results in

```
ls = [i for i in range(10)]
```

with all comments below the code deleted.

**Comment by Hellebore** [comment]
Thanks for raising! Have reproduced and will look into fixing.

**Comment by brendanator** [comment]
This happens inside any function/method which contains indented comments after the end of the code

**Comment by Hellebore** [comment]
This should be addressed by the next release.

**Comment by Hellebore** [comment]
Fixed with release of 0.11.1
